### PR TITLE
Madison dev

### PR DESCRIPTION
**Pull request by MadisonFichtner**
-Refactored CSS with new CSS files for each corresponding HTML page
-Updated sidebar menu to show appropriate options for authorized and unauthorized users
-Added icons to sidebar menu
-Added footer items to side menu
-User is now logged out if the browser is closed, or if they have been idle
-Displays logged in users name alongside the logout button

### CODE REVIEW

File: psychiatric_guide_app/settings.py [hunk 1]
```diff
@@ -143,8 +143,12 @@
 # https://docs.djangoproject.com/en/2.1/howto/static-files/
 
 STATIC_URL = '/static/'
-STATIC_ROOT = 'static'
+STATIC_ROOT = ''
```
Comment (erynplant) [inline psychiatric_guide_app/settings.py]: this needs to be set as 'static' for app engine

Comment (MadisonFichtner) [reply psychiatric_guide_app/settings.py]: Added: 'static' to STATIC_ROOT

File: application/templates/application/login.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
         {% csrf_token %}
         {{ form|crispy }}
         <div style="text-align: center;">
-            <button class="btn btn-primary mt-4 btn-block" type="submit">Login</button>
+            <button class="btn btn-primary mt-4 btn-block " type="submit">Login</button>
```
Comment (erynplant) [inline application/templates/application/login.html]: no

File: application/templates/application/global-html.html [hunk 3]
```diff
@@ -35,47 +42,95 @@
...
+                    <div class="dropdown">
+                        <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton"
+                                data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    Hello {{ user.first_name }}.    <!--shows users first name if logged in.--><!--
+                        </button>
+                        <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+                            <a class="dropdown-item" href="{% url 'logout-view' %}">Logout</a>
+                        </div>
+                    </div>
+                {% endif %}-->
```
Comment (erynplant) [inline application/templates/application/global-html.html:58]: is this something that we want to add later or can we just get rid of it?

File: application/templates/application/global-html.html [hunk 4]
```diff
@@ -133,7 +188,7 @@ <h3>Psychiatric Guide</h3> <!-- Generic Title -->
         </div>
     </center>
 </footer>
-
+-->
```
Comment (erynplant) [inline application/templates/application/global-html.html:197]: Same here. All of this is getting handled in the sidebar so we should just get rid of it.

**Review by erynplant — CHANGES_REQUESTED** [review]

File: application/templates/application/login.html (hunk 2 above)
Comment (erynplant) [reply application/templates/application/login.html]: yes

**Pull request merged**